config/routes.rb

```diff
@@ -6,12 +6,17 @@
   resources :availabilities, only: [:index] do
     collection do
       post '/', to: 'availabilities#update_availabilities'
+      get '/default', to: 'availabilities#default_index'
+      get '/default/places/:id', to: 'availabilities#default_edit'
```

Comment:
If you need the ID put it in `member do` and not `collection do`

Also maybe this is helpful: http://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing

Reply:
@donjar So maybe sth like
```ruby
collection do
  resources :places, only: %i[index edit update]
end
```
But how to make those resources point to different controller actions (that is, I want a distinct index as compared to the GET /availabilities

Reply:
Because user would specify the same availabilities for all places but the default user would be different in different places